### ISSUE TITLE
Untargetable Razor?

### ISSUE THREAD

**Issue by GoogleFrog** [ZeroK-RTS/Zero-K#4790]
https://zero-k.info/Battles/Detail/1510265

**Comment by sprunk** [comment]
![image](https://user-images.githubusercontent.com/2573076/210919922-1c221860-7951-4601-ab57-db4d6bd20819.png)

There was a commander wreck mostly covering the Razor, or perhaps its center. The Razor was targetable when a unit positioned itself well (as depicted) but this seemed very hard to achieve and fragile. Removing the wreck via `/godmode` confirmed units could now shoot the Razor normally.

**Comment by GoogleFrog** [comment]
Fix comm wreck hitvolume.

**Comment by sprunk** [comment]
Yes, though the general case might not be solveable (what if a Krow crashed into it?).
